gen/libdir/forbidden.conf

```diff
@@ -45,11 +45,161 @@
+# let ca = fr | co
+# let Exp-obs =
+#   [Exp & M]; rf & ext; [Exp & M]
+#   | [Exp & M]; ca & ext; [Exp & M]
+-safe Rfe Fre Coe
+
+#let Exp-haz-ob = [Exp & R]; (po & same-loc); [Exp & R]; (ca & ext); [Exp & W]
+-safe [PosRR Coe] [PosRR Fre]
+
+### Data and address dependance
+# what is same instance?
+# let data = [Exp & R]; (basic-dep; [DATA]; iico_data+; [Exp & W]) & ~same-instance
+# let dob = data
+-safe DpDatadW DpDatasW
+
+# let addr = [Exp & R]; (basic-dep; [ADDR]; iico_data+; [Exp & M | Imp & Tag & R | Imp & TTD & R | HU | TLBI | DC.CVAU | IC.IVAU]) & ~same-instance
+# let dob = addr
+-safe DpAddrdR DpAddrdW DpAddrsR DpAddrsW
+
+# let ctrl = [Exp & R]; basic-dep; [BCC]; po
+# let dob = ctrl; [Exp & W | HU | TLBI | DC.CVAU | IC]
+# mcat2config7 aarch64.cat -let dob
+-safe DpCtrldW DpCtrlsW
+
+# let dob = addr; [Exp & M]; po; [Exp & W | HU]
+-safe [DpAddrdW PodWW] [DpAddrdW PosWW] [DpAddrsW PodWW] [DpAddrsW PosWW] [DpAddrdR PodRW] [DpAddrdR PosRW] [DpAddrsR PodRW] [DpAddrsR PosRW]
```

Comment:
Not for this patch, but it seems it might be useful to add support for `DpAddrd*` `DpAddrs*` and similarly for `DpCtrld*` and `DpCtrls*` and this could be surprised by both diyone7 and diy7